### DISCUSSION TITLE
Enable to install it in QNAP 470 PRO

### DISCUSSION THREAD

**Discussion by abdoubetta** [open-webui/open-webui#5274] [Q&A]
Hello,
 I'm using open-webui on my MacBook Air without any issue. But When I tried to installed in on QNAP server using this command :
docker pull ghcr.io/open-webui/open-webui:main

I got this error  : 
Error response from daemon: manifest for ghcr.io/open-webui/open-webui:main not found

Note that docker is working well in my QNAP and I was able to install other docker packages.

**Comment by justinh-rahb** [comment]
QNAP Container Station is known to be buggy with container registries other than Docker Hub. You can try adding https://ghcr.io to the registries.
